docs/wasm-wasi-on-kubernetes.md

```diff
@@ -1,7 +1,7 @@
 # Running wasi workload natively on kubernetes using crun
 
-Crun natively supports running wasm/wasi workload on using `wasmedge/wasmer`.
-Both `wasmedge/wasmer` comes with their own set of unique features.
+Crun natively supports running wasm/wasi workload on using `wasmedge, wasmer and wasmtime`.
+Each one of them (`wasmedge, wasmer and wasmtime`) comes with their own set of unique features.
```

Comment:
nit: should we quote each of them like `wasmedge`, `wasmer` and `wasmtime`? (both two occurrences)

Reply:
fixed.